src/priorityqueue.jl

```diff
@@ -324,6 +324,29 @@ function next(pq::PriorityQueue{K,V}, i) where {K,V}
     return (pq.xs[idx], i)
 end
 
+"""
+    delete!(pq, key)
+
+Delete the mapping for the given key in a priority queue, and return the priority queue.
+# Examples
+```jldoctest
+julia> q = PriorityQueue(["a","b","c"],[2,3,1],Base.Order.Forward)
```

Comment:
`PriorityQueue`s no longer accept two iterables during construction (matching the corresponding change in other data structures like `Dict`s).  Some alternatives include

```
q = PriorityQueue(Base.Order.Forward, "a"=>2, "b"=>3, "c"=>1)
q = PriorityQueue(Base.Order.Forward, zip(["a","b","c"],[2,3,1]))
q = PriorityQueue(zip(["a","b","c"],[2,3,1]), Base.Order.Forward)
q = PriorityQueue(Base.Order.Forward, [("a", 2), ("b", 3), ("c", 1)])
```

(The same change needs to be made in the docstring for the `struct`, if you're up for it.)